readium/navigator/src/main/java/org/readium/r2/navigator/HyperlinkNavigator.kt

```diff
@@ -16,6 +17,19 @@ import org.readium.r2.shared.util.AbsoluteUrl
 @ExperimentalReadiumApi
 public interface HyperlinkNavigator : Navigator {
 
+    @ExperimentalReadiumApi
+    public sealed interface LinkContext {
+        public val referrer: String
+        public val activationPoint: PointF
```

Comment:
What happens if we activate a link using a keyboard interaction? Maybe we should have an `activationEvent: ActivationEvent?` with `{ KeyboardEvent, TapEvent(PointF) }`?

Reply:
That's not possible at the moment, is it? Or at least, it would not be intercepted. Actually, I may just have picked up a wrong name. I believe the point of interest is the location of the noteref, because the popup should be anchored there.